xwiki-platform-core/xwiki-platform-resource/xwiki-platform-resource-api/src/main/java/org/xwiki/resource/ResourceType.java

```diff
@@ -35,12 +35,27 @@
      */
     private String id;
 
+    /**
+     * True if this type represents a static resource.
```

Comment:
This should explain what is a static resource as otherwise it's useless :) (and it would be good to know what is a static resource here)

Reply:
Actually you should define that in `isStatic()` and link to it.